.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
why did you add this?
I don't wont to get mails for every failure.
The default works quite fine for me.